common/djangoapps/util/milestones_helpers.py

```diff
@@ -313,6 +315,16 @@ def remove_content_references(content_id):
     return milestones_api.remove_content_references(content_id)
 
 
+def any_unfulfilled_milestones(course_id, user_id):
+    """ Returns a boolean if user has any unfulfilled milestones """
+    try:
+        return bool(
+            get_course_milestones_fulfillment_paths(course_id, {"id": user_id})
+        )
+    except InvalidMilestoneRelationshipTypeException:
```

Comment:
I don't recall seeing this exception being caught but if it's being raised then I figured it would need to be caught so we decided to catch it in here instead of in utils.py